src/py21cmfast/inputs.py

```diff
@@ -315,6 +315,12 @@ class GlobalParams(StructInstanceWrapper):
     AVG_BELOW_SAMPLER:
         int flag to add the average halo proeprties in each cell between the source mass limit
         and the mass sampled by the halo sampler
+    UPPER_STELLAR_TURNOVER_MASS:
+        The pivot mass associated with the optional upper mass power-law of the stellar-halo mass relation
+        (see FlagOptions.USE_UPPER_STELLAR_TURNOVER)
+    UPPER_STELLAR_TURNOVER_INDEX:
+        The power-law index associated with the optional upper mass power-law of the stellar-halo mass relation
+        (see FlagOptions.USE_UPPER_STELLAR_TURNOVER)
```

Comment:
Is there a good reason these aren't in `AstroParams`? Eventually we want to move as many global params out as possible

Reply:
Yeah, I think we should move towards having ~no global parameters. However, we should think about this further in #9 and what criteria we want to have to potentially keep something as a global. But for now, let's assume we make all parameter-like things non-global.

Reply:
They've been moved to `AstroParams` now